projects/lf-documentation/src/app/getting-started-cdn-example/getting-started-cdn-example.component.html

```diff
@@ -0,0 +1,163 @@
+<div class="toc-container">
+  <p class="toc-container-header"> Contents</p>
+  <ul>
+    <li>
+      <a href="#cdn-example-angular">Example: <code>lf-checklist</code> In an Angular Project Via CDN</a>
+    </li>
+    <li>
+      <a href="#cdn-example-react">Example: <code>lf-checklist</code> In a React Project Via CDN</a>
+    </li>
+  </ul>
+</div>
+<h3 id="cdn-example-angular"> Example: <code>lf-checklist</code> In an Angular Project Via CDN</h3>
+<p>In the example below, the implementation comes from the CDN and the types are provided
+  in the <code>@laserfiche/types-lf-ui-components</code> NPM package.</p>
+<ol>
+  <li>
+    <p  class="documentation-text">
+      Add the following script tags to <code>index.html</code>
+    </p>
+    <pre class="code-container">
+<span class="code-html-tagname"> &lt;script </span><span class="code-html-attribute-name">src</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/laserfiche-ui-components.js"</span><span class="code-html-attribute-name"
+> defer</span><span class="code-html-tagname">&gt;&lt;/script&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/lf-laserfiche-lite.css"</span><span class="code-html-attribute-name"> rel</span><span
+>=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
```

Comment:
I think there should still be the line 
`If you are not using Angular Material, you must add an Angular Material stylesheet.`

(separate from the lf-laserfiche-lite)